### DISCUSSION TITLE
Robot Reviewer Down?

### DISCUSSION THREAD

**Discussion by kdaly3** [community/community#37902] [New to GitHub]
Hi I am a PhD student that was utilizing the risk of bias program a few years ago, I came back to redo a part of my review, and can't get the tool to load? Is it no longer accessible? Can anyone help by any chance? Thanks!

**Comment by jfullstackdev** [comment]
what is that particular project or file you want to view ?

**Comment by kdaly3** [comment]
I was trying to repeat a risk of bias assessment for 11 articles I am
reviewing for a systematic review.
